packages/server/lib/util/file.js

```diff
@@ -2,7 +2,7 @@ const _ = require('lodash')
 const os = require('os')
 const md5 = require('md5')
 const path = require('path')
-const debug = require('debug')('cypress:server:file')
+const debug = require('debug')('cypress-verbose:server:file')
```

Comment:
For consistency, should we name this `debugVerbose` even though there are no non-verbose debugs in here?

Reply:
Agreed